Dockerfile

```
@@ -0,0 +1,22 @@
+FROM node:21-bookworm AS base
```

Might be worth picking the alpine LTS version?

```suggestion
FROM node:20-alpine AS base
```

---

do we wanna go bleeding edge yolo and go to node:22-alpine?

---

Saw this comment and made a [fix to setup-app](https://github.com/safeinsights/setup-app/pull/2) to use the `alpine` variant for LTS there too :sweat_smile: